waspc/src/Wasp/Psl/Util.hs

```diff
@@ -1,31 +1,39 @@
 module Wasp.Psl.Util where
 
 import Data.Foldable (find)
-import qualified Wasp.Psl.Ast.Model as PslModel
+import qualified Wasp.Psl.Ast.Argument as Psl.Argument
+import qualified Wasp.Psl.Ast.Attribute as Psl.Attribute
+import qualified Wasp.Psl.Ast.ConfigBlock as Psl.ConfigBlock
+import qualified Wasp.Psl.Ast.Model as Psl.Model
 
-findIdField :: PslModel.Body -> Maybe PslModel.Field
-findIdField (PslModel.Body elements) = find isIdField fields
+findIdField :: Psl.Model.Body -> Maybe Psl.Model.Field
+findIdField (Psl.Model.Body elements) = find isIdField fields
   where
-    fields = [field | (PslModel.ElementField field) <- elements]
+    fields = [field | (Psl.Model.ElementField field) <- elements]
 
-    isIdField :: PslModel.Field -> Bool
-    isIdField PslModel.Field {_attrs = attrs} = any (\attr -> PslModel._attrName attr == attrNameAssociatedWitIdField) attrs
+    isIdField :: Psl.Model.Field -> Bool
+    isIdField Psl.Model.Field {_attrs = attrs} = any (\attr -> Psl.Attribute._attrName attr == attrNameAssociatedWitIdField) attrs
 
     -- We define an ID field as a field that has the @id attribute.
     attrNameAssociatedWitIdField :: String
     attrNameAssociatedWitIdField = "id"
 
-findIdBlockAttribute :: PslModel.Body -> Maybe PslModel.Attribute
-findIdBlockAttribute (PslModel.Body elements) = find isIdBlockAttribute attributes
+findIdBlockAttribute :: Psl.Model.Body -> Maybe Psl.Attribute.Attribute
+findIdBlockAttribute (Psl.Model.Body elements) = find isIdBlockAttribute attributes
   where
-    attributes = [attr | (PslModel.ElementBlockAttribute attr) <- elements]
+    attributes = [attr | (Psl.Model.ElementBlockAttribute attr) <- elements]
 
-    isIdBlockAttribute :: PslModel.Attribute -> Bool
-    isIdBlockAttribute PslModel.Attribute {_attrName = attrName} = attrName == idBlockAttributeName
+    isIdBlockAttribute :: Psl.Attribute.Attribute -> Bool
+    isIdBlockAttribute Psl.Attribute.Attribute {_attrName = attrName} = attrName == idBlockAttributeName
 
     -- We define the ID block attribute as an attribute with the name @@id.
     idBlockAttributeName :: String
     idBlockAttributeName = "id"
 
-doesPslFieldHaveAttribute :: String -> PslModel.Field -> Bool
-doesPslFieldHaveAttribute name PslModel.Field {_attrs = attrs} = any ((== name) . PslModel._attrName) attrs
+doesPslFieldHaveAttribute :: String -> Psl.Model.Field -> Bool
+doesPslFieldHaveAttribute name Psl.Model.Field {_attrs = attrs} = any ((== name) . Psl.Attribute._attrName) attrs
+
+findPrismaConfigBlockKeyValuePair :: String -> [Psl.ConfigBlock.KeyValuePair] -> Maybe Psl.Argument.Expression
```

Comment:
Similar kere, could this `Strign` be replaceable with one of `type`s we introduced? If not, that is ok, I am not sure myself.